provider/ansible.py

```diff
@@ -132,6 +133,12 @@ def python_install():
             LOG_JOB.error("Failed to get available pip binary")
             return False
         install_cmd = '%s install ansible' % pip_bin
+        # On s390x and ppc64le, the ansible dependency cryptography needs
+        # Rust >=1.41.0, which can only be installed by yum
+        if arch.ARCH in ('s390x', 'ppc64le'):
+            if not utils_package.package_install('rust cargo'):
```

Comment:
pkgs should be a list.

Reply:
@PaulYuuu, updated. Thanks. I define 'package_list' in subtests-internal.cfg by submitting a new patch https://gitlab.cee.redhat.com/kvm-qe/kar/-/merge_requests/853

Test PASS:
 (01/11) Host_RHEL.m8.u6.v0.product_rhel.nographic.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.6.0.s390x.io-github-autotest-qemu.unattended_install.cdrom.extra_cdrom_ks.default_install.aio_threads.s390-virtio: STARTED
 (01/11) Host_RHEL.m8.u6.v0.product_rhel.nographic.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.6.0.s390x.io-github-autotest-qemu.unattended_install.cdrom.extra_cdrom_ks.default_install.aio_threads.s390-virtio:  PASS (580.76 s)
 (02/11) L1_deployment.Host_RHEL.m8.u6.v0.product_rhel.nographic.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.6.0.s390x.io-github-autotest-qemu.ansible_test.s390-virtio: STARTED
 (02/11) L1_deployment.Host_RHEL.m8.u6.v0.product_rhel.nographic.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.6.0.s390x.io-github-autotest-qemu.ansible_test.s390-virtio:  PASS (554.40 s)

@qcheng-redhat Could you also have a review since there are some common code change? Thanks in advance.